<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video1" />
    <param name="camera_info_url" type="string" value="file://$(find usb_cam)/cfg/head_camera.yaml"/>
    <param name="image_width" value="320" />
    <param name="image_height" value="240" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="framerate" value="5" />
    <param name="pixel_format" value="yuyv" />
  </node>
  <node name="ar_pose" pkg="ar_pose" type="ar_multi" respawn="false" output="screen">
    <param name="marker_pattern_list" type="string" value="$(find usb_cam)/patterns/column_markers"/>
    <param name="threshold" type="int" value="80"/>
    <param name="publish_tf" type="bool" value="true"/> 
  </node>
</launch>
